README.md

```diff
@@ -62,6 +62,61 @@ docker run -v /path/to/my-code:/src/main ponylang/ponyc ./main
 
 If you're using `docker-machine` instead of native docker, make sure you aren't using [an incompatible version of Virtualbox](#virtualbox).
 
+## Linux using an RPM package (via Bintray)
+
+For Red Hat, CentOS, or Fedora Linux, the `master` and `monthly` branches are packaged and availabe on Bintray ([pony-language/ponyc-rpm](https://bintray.com/pony-language/ponyc-rpm)).
+
+To install a single RPM from Bintray:
+```bash
+wget https://dl.bintray.com/pony-language/ponyc-rpm/ponyc-VERSION.rpm
+sudo rpm --install ponyc-VERSION.rpm
+yum info ponyc
+ponyc --version
+```
+
+To install monthly / release builds via Yum:
+```bash
+wget https://bintray.com/pony-language/ponyc-rpm/rpm -O bintray-pony-language-ponyc-rpm.repo
+sudo mv bintray-pony-language-ponyc-rpm.repo /etc/yum.repos.d/
+
+sudo yum install ponyc
+```
+
+To install master builds via Yum:
+```bash
+wget https://bintray.com/pony-language/ponyc-rpm/rpm -O bintray-pony-language-ponyc-rpm.repo
+sudo mv bintray-pony-language-ponyc-rpm.repo /etc/yum.repos.d/
+
+sudo yum install yum-utils
+repoquery --show-duplicates ponyc
+
+yum install ponyc-master_0.2.1_49_b15a0c5
+```
+
+That last bit of versioning is where you'll notice that we're not quite matching the name-version-release-arch pattern [described here for Fedora](https://fedoraproject.org/wiki/Packaging:Naming?rd=Packaging:NamingGuidelines#Non-Numeric_Version_in_Release), but things should work anyways for now...
+
+
+<!---
+## Linux using a DEB package (via Bintray)
+
+For Ubuntu or Debian Linux, the `master` and `monthly` branches are packaged and availabe on Bintray ([pony-language/ponyc-debian](https://bintray.com/pony-language/ponyc-debian)) *(although this is broken until we further clean up version numbering within the packages)*:
```

Comment:
Debian, however, is broken right now even in the basic case:
![screen shot 2016-08-02 at 12 50 50 pm](https://cloud.githubusercontent.com/assets/195060/17342822/cc6dc812-58af-11e6-96e4-9cdc2389eb0d.png)

🐒